journal/modernized/42-appendix.adoc

```diff
@@ -797,6 +800,7 @@ On the 3rd of Eleventh Month,
 we started in company with C. Barff and a little boy on an
 excursion to the famous marai of Tabutabu Atea,--situated in
 // lint-disable invalid-character
+// I'm not sure what that line above is. "marai" in 799 in the book is underlined, not sure what it means. 
```

Comment:
I remember that there were a bunch of "islander" terms that were used in the latter part of this book. I think this is one of them....maybe the name for the place where the islanders did human sacrifice previously. I would leve it.